hera_cal/smooth_cal.py

```diff
@@ -67,6 +68,84 @@ def single_iterative_fft_dly(gains, wgts, freqs, conv_crit=1e-5, maxiter=100):
     return np.sum(taus)
 
 
+def dpss_filters(freqs, times, freq_scale=10, time_scale=1800, eigenval_cutoff=1e-9, return_1D_filters=False):
+    """Generate a set of 2D discrete prolate spheroidal sequence (DPSS) filters
+    to filter calibration solutions along the time and frequency axes simultaneously.
+
+    Arguments:
+        freqs: ndarray of frequency channels in Hz
+        times: ndarray of shape=(Ntimes) of Julian dates as floats in units of days
+        freq_scale: frequency scale in MHz to use for the low-pass filter. freq_scale^-1 corresponds
+            to the half-width (i.e. the width of the positive part) of the region in fourier
+            space, symmetric about 0, that is retained after filtering.
+            Note that freq_scale is in MHz while freqs is in Hz.
+        time_scale: time scale in seconds. Defined analogously to freq_scale.
+            Note that time_scale is in seconds, times is in days.
+        eigenval_cutoff: sinc_matrix eigenvalue cutoff to use for included dpss modes.
+            Only used when the filtering method is 'DPSS'
+        return_1D_filters: if True, return the 1D filters that make up the 2D DPSS filter.
+            Default is False
+
+    Returns:
+        filters: DPSS filtering vectors, ndarray of size (Ntimes * Nfreqs, N_frequency_vectors * N_time_vectors)
```

Comment:
Explain how the output changes if return_1D_filters is True

Reply:
I ended up removing this when I simplified the `CalbrationSmoother.time_freq_2d_filter` function